agiocli/utils.py

```diff
@@ -303,6 +303,7 @@ def parse_project_string(user_input):
     asstype_abbrev = match.group("asstype").lower()
     if asstype_abbrev not in assignment_types:
         asstypes = ", ".join(assignment_types.keys())
+        return False
         sys.exit(
             f"Error: unsupported assignment type: '{asstype_abbrev}'.  "
             f"Recognized shortcuts: {asstypes}"
```

Comment:
Not sure what to do about this error message now.

Reply:
Should we throw an exception and catch it in the caller?